pkgs/data/documentation/nixpkgs-manual/options-doc.nix

```diff
@@ -0,0 +1,32 @@
+# NB: This file describes the Nixpkgs manual, which happens to use module docs infra originally developed for NixOS.
+{ lib, nixosOptionsDoc }:
+
+let
+  inherit (lib.strings) hasPrefix removePrefix;
+
+  modules = lib.evalModules {
+    modules = [ ../../../top-level/config.nix ];
```

Comment:
We could use dependency injection for this module and for the root path as well. Less predictions would make it more portable.

Reply:
Agree. I'll do that separately, I think.